src/NECompletion/NECUsageTracker.class.st

```diff
@@ -0,0 +1,27 @@
+"
+Tracks how often completion entries are used
+"
+Class {
+	#name : 'NECUsageTracker',
+	#superclass : 'Object',
+	#classVars : [
+		'UsageCounts'
```

Comment:
There should be a class side #reset Method to set the class var back to nil. We could call it for runtime image cleaning/size reduction

```Smalltalk
reset
     <script>
     UsageCounts := nil
```